experimenter/experimenter/nimbus-ui/src/components/PageSummary/index.test.tsx

```diff
@@ -956,7 +956,7 @@ describe("PageSummary", () => {
     render(<Subject mocks={[mockRollout]} />);
     await waitFor(() =>
       expect(
-        screen.queryByTestId("desktop-min-version-warning"),
+        screen.queryByTestId("warning-desktop-min-version"),
       ).toBeInTheDocument(),
```

Comment:
nit: replace the `expect(...).toBeInTheDocument()` with `screen.getByTestId(...)`

`getByTestId` will throw if it does not find the selector.